### PR TITLE
Feature/add legend montoring

### PR DESCRIPTION
**Pull request by GabrielMoreschi**
Foi adicionado o estilo dos polígonos de monitoramento e a respectiva legenda.

### CODE REVIEW

File: components/monitoring/MonitoringLayers.vue [hunk 1]
```diff
@@ -162,18 +212,18 @@ export default {
...
+                    style.color = '#ff3333'
                     break
-                case 'DR':
-                    style.color = '#337f1e'
+                case 'Degradação':
+                    style.color = '#ff8000'
                     break
-                case 'FF':
-                    style.color = '#ba1a1a'
+                case 'Fogo em FLoresta':
```
Comment (dagnaldo) [inline components/monitoring/MonitoringLayers.vue:222]: `FLoresta? `

File: pages/index/monitoring.vue [hunk 2]
```diff
@@ -34,6 +34,28 @@
...
+        <div v-if="showFeatures" class="px-4">
+            <p class="font-weight-regular pt-2">Legenda:</p>
+            <v-col>
+                <v-row class="mb-2">
+                    <v-icon class="mr-2" color="#990099">mdi-square</v-icon>
+                    Desmatamento em Regeneração
+                </v-row>
+                <v-row class="mb-2">
+                    <v-icon class="mr-2" color="#b35900">mdi-square</v-icon>
+                    Fogo em FLoresta
```
Comment (dagnaldo) [inline pages/index/monitoring.vue:46]: Why dont you call it from a `data object` on vue?

Comment (dagnaldo) [reply pages/index/monitoring.vue:46]: Take a look, it is with `FLoresta`

**Review by dagnaldo — CHANGES_REQUESTED** [review]

File: components/monitoring/MonitoringLayers.vue (hunk 1 above)
Comment (GabrielMoreschi) [reply components/monitoring/MonitoringLayers.vue]: Resolved

File: pages/index/monitoring.vue (hunk 2 above)
Comment (GabrielMoreschi) [reply pages/index/monitoring.vue]: I'm following de legend pattern of the other components.

**Review by dagnaldo — APPROVED** [review]

**Pull request merged**